src/Components/NavBar/NavBar.tsx

```diff
@@ -471,6 +491,13 @@ export const NavBar: React.FC<React.PropsWithChildren<unknown>> = track(
   )
 })
 
+// Hide these links earlier to ensure "Collecting 101" has space on smaller widths
+
+const NavBarItemFairsLink = styled(NavBarItemLink)`
+  @media (max-width: 1000px) {
+    display: none;
+  }
+`
```

Comment:
Any reason we're not using Palette's responsive props? For the most part, any time custom CSS is used it can be regarded as an anti-pattern which will lead to other problems:

https://palette.artsy.net/guides/responsive/

Reply:
See [here](https://github.com/artsy/force/blob/main/src/Components/NavBar/NavBar.tsx#L214) for an example -- on mobile, display none. If we're concerned about the next breakpoint up, its something like 

```
display={["none", "none", "flex"]}
```

(Not sure if this the exact way to handle the behavior in this case, but using palette (somehow) is the way)

Reply:
@damassi we are using responsive props above in the Flex boxes around the other nav items. This was the pattern already laid out specifically for Museums, so I followed a similar pattern. Are you saying make a new flex box around these 2 cases instead?

Reply:
Ah, if its already something that's laid out pattern wise all good and non-blocking -- (tho it would be interesting to see if the responsive props like in the above would solve both of those nav items while also improving the code. Worth a shot)

Reply:
But if stuff gets weird, no prob at all 👍

Reply:
Yeah, the reason is that we can precisely hide them this way. Simply measure the exact point it starts to clip.